src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -77,3 +80,69 @@ def get_initial_value(self, submission: Submission | None = None):
         if submission is None:
             return None
         return submission.payments.get_completed_public_order_ids()
+
+
+@register("cosign")
```

Comment:
```suggestion
@register("cosign_data")
```
let's use a proper variable name :)